setup.py

```diff
@@ -22,7 +22,7 @@
 ]
 
 setup(
-    name='aldryn-redirects',
+    name='aldryn-redirects-fil',
     version=__version__,
     description='A modified version of django.contrib.redirects with multilingual target URLs.',
     author='Divio AG',
```

Comment:
You'll need to change the author!

Reply:
Changed the author. Thanks

Reply:
You may want to change the title of the PR to something more meaningful so that the commit message looks good, something like: "feat: Change the package name to aldryn-redirects-fil"

Reply:
Done

Reply:
@vipulnarang95 I meant the PR title not the description, when you merge you need to male sure it's squashed and the commit message is the same as the PR title:

![Screenshot from 2023-02-14 11-05-33](https://user-images.githubusercontent.com/12543977/218719163-67aaeb65-7aea-4c50-b3ff-9130f2fc07f6.png)